src/VoyagerHooksServiceProvider.php

```diff
@@ -13,20 +13,41 @@ class VoyagerHooksServiceProvider extends ServiceProvider
      */
     public function register()
     {
+        if (config('voyager-hooks.disable', false) {
+            return;
+        }
+            
+
+        if ($this->app->runningInConsole()) {
+            $config = dirname(__DIR__) . '/publishable/config/voyager-hooks.php';
+
+            $this->publishes(
+                [$config => config_path('voyager-hooks.php')],
+                'Voyager-hooks config'
+            );
+        }
+
+
         // Register the HooksServiceProvider
         $this->app->register(HooksServiceProvider::class);
 
         // Load views
-        $this->loadViewsFrom(__DIR__.'/../resources/views', 'voyager-hooks');
+        $this->loadViewsFrom(__DIR__ . '/../resources/views', 'voyager-hooks');
     }
 
     /**
      * Bootstrap the application services.
      *
      * @param \Illuminate\Events\Dispatcher $events
+     *
+     * @return bool
      */
     public function boot(Dispatcher $events)
     {
+        if (config('voyager-hooks.disable', false) {
+            return true;
```

Comment:
Is there a reason to `return true;` instead of just `return;`? The former requires actually returning a bool, where as the latter will just exit the function early.

Reply:
Corrected